src/libraries/System.Net.NameResolution/tests/FunctionalTests/GetHostAddressesTest.cs

```diff
@@ -174,19 +174,33 @@ public async Task DnsGetHostAddresses_PreCancelledToken_Throws()
         [OuterLoop]
         [Fact]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/43816")] // Race condition outlined below.
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/33378", TestPlatforms.AnyUnix)] // Cancellation of an outstanding getaddrinfo is not supported on *nix.
         public async Task DnsGetHostAddresses_PostCancelledToken_Throws()
         {
-            using var cts = new CancellationTokenSource();
+            const int numberOfRequests = 100;
```

Comment:
Nit: PascalCasing

Reply:
Within a method too? Just want to confirm (this is new to me).

Reply:
Yup.  `const` == PascalCasing, regardless of where it's used. :)